docs/source/certs_management.md

```diff
@@ -0,0 +1,381 @@
+
+#### Enrollment Certificates
+
+Enrollment certificates are classed into four types:
+
+* **Admin**
+* **Peer**
+* **Orderer**
+* **Client**
+
+Each enrollment certificate type has a specific role:
+
+**Admin:** X.509 certificates used to authenticate admin identities, which are required to make changes to  Fabric configurations.
+
+**Peer:** X.509 certificates used to enroll peer nodes, located physically on the node or mapped to the node. For a Fabric peer node to start, it must have a valid enrollment certificate with the required attributes.
+
+**Orderer:** X.509 certificates used to enroll orderer nodes, located physically on the node or mapped to the node. For a Fabric orderer node to start, it must have a valid enrollment certificate with the required attributes.
+
+**Client:** X.509 certificates that allow signed requests to be passed from clients to Fabric nodes. These requests require a signature for admin level certificates.
+
+
+#### TLS Certificates
+
+TLS certificates allow Fabric nodes and clients to sign and encrypt communications. A valid TLS Certificate is required for any channel communication.
+
+
+### Enrollment Certificate Expiration
+
+All Enrollment certificates are assigned an expiration date by the issuing Certificate Authority (CA).  Expiration dates must be monitored, and certificates must be re-enrolled before expiration. The most important Enrollment Certificate parameter is the **Not After** element, which indicates its expiration date.
+
+## Certificates and Locations
+
+Organization CAs supply X.509 Enrollment Certificates for identities and the TLS CAs supply TLS Certificates for securing node and client communications.
+
+
+### Organization CA Certificates
+
+Organization CA Root certificates and Organization CA Admin certificates are issued for each organization.
+
+#### Organization CA Root Certificate
+
+**Description**: Public certificate that permits verification of all certificates issued by the Organization CA.
+
+**Location**: Stored on disk in the Organization CA directory, or in the channel configuration (ca-cert.pem).
+
+**Impact if expired**: A new Organization CA Root certificate must be issued. Organization CA Root certificates are valid for 15 years.
+
+
+#### Organization CA Admin Certificate
+
+**Description**: Certificate used when making admin requests to the Organization CA.
+
+**Location**: Dependent on implementation:
+
+<pre>
+msp
+ ├── IssuerPublicKey
+ ├── IssuerRevocationPublicKey
+ ├── cacerts
+ │   └── localhost-7053.pem
+ ├── keystore
+ │   └── key.pem
+ ├── signcerts
+ │   └── cert.pem
+ └── user
+</pre>
+
+**Impact if expired**: Cannot register or enroll new identities, but transaction traffic does not stop.
+
+[Reference](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-orderer-org-s-ca-admin)
+
+
+### TLS CA Certificates
+
+TLS CA Root certificates and TLS CA Admin Certificates are issued for each organization.
```

Comment:
All the comments above for Organization CA Certificates also applies to TLS CA Certificates.

Reply:
ok.